config/initializers/merit.rb

```diff
@@ -21,10 +21,17 @@
 
 # Create application badges (uses https://github.com/norman/ambry)
 
-Merit::Badge.create!(
-  id: 1,
-  name: 'checked_in'
-)
+badges = [  {id: 1, name: 'checked_in', custom_fields: {image: "/shooting-star.png"} }, {id: 2, name: 'checked_inx2', custom_fields: {image: "/ruby heart.png"} }]
```

Comment:
Line is too long. [162/80]<br>Space inside square brackets detected.<br>Space inside { missing.<br>Space inside } missing.